.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+go:
```

Comment:
There was a cgo compilation failure on go 1.5 so I ripped it out, for now.
The Go community seems pretty up to date so I dont think we need to support older versions until people ask us to.